app/marketing/mails.py

```diff
@@ -1152,7 +1152,14 @@ def new_bounty_daily(bounties, old_bounties, to_emails=None):
         try:
             setup_lang(to_email)
             from_email = settings.CONTACT_EMAIL
-            html, text = render_new_bounty(to_email, bounties, old_bounties)
+
```

Comment:
are you planning on updating the subject line of this email? right now it advertises new bounties. it should probalby not anymore. 

@frankchen07 any thoughts on a great subject line?

Reply:
Suggestion: (The?) **Gitcoin Daily Digest**

Reply:
@PixelantDesign @frankchen07 any thoughts here? Gitcoin Daily Digest is good but rather generic. i wonder if we should test adding dynamic data to it.

Reply:
Lets go with "Gitcoin Daily (x Bounties, y quests/ z hacakthons)" where each variable is the number of each in the email

Reply:
Done in [03137b3](https://github.com/gitcoinco/web/pull/6471/commits/03137b362bf6ab1f3e86198bd69db65269b9e8d3)